src/llscan.cc

```diff
@@ -241,7 +233,19 @@ bool FindInstancesCmd::DoExecute(SBDebugger d, char** cmd,
     }
 
   } else {
-    result.Printf("No objects found with type name %s\n", type_name.c_str());
+    Settings* settings = Settings::GetSettings();
+    if (settings->ShouldUseColor()) {
+      rang::setControlMode(rang::control::Force);
+    } else {
+      rang::setControlMode(rang::control::Off);
+    }
```

Comment:
This code is used frequently. It should probably be converted into a MACRO or inlined function. Or maybe we can call "rang::setControlMode" on SetPropertyColorCmd::DoExecute? I'm guessing this would be persistent across other commands, and also call it on `llnode::PluginInitialize`.